pkg/diagnostics/networkpod/util/util.go

```diff
@@ -31,6 +31,8 @@ const (
 	NetworkDiagNodeLogDirPrefix      = "/nodes"
 	NetworkDiagMasterLogDirPrefix    = "/master"
 	NetworkDiagPodLogDirPrefix       = "/pods"
+
+	NetworkDiagDefaultPodImage = "openshift/origin"
```

Comment:
We are not dependent on exact image, any image that provides shell functionality will work (openshift/origin, openshift3/origin, rhel7). We have embedded the actual script in the pod definition as clayton suggested. 
Yes, user can override the image. Diagnostics takes a config flag `oadm diagnostics NetworkCheck --network-pod-image=openshift3/origin` 

I see your point. openshift/origin will not be available in OCP registry and if the user didn't use the pod image override flag then this will not work. I'm assuming `variable.DefaultImagePrefix` will be openshift/origin for origin release and openshift3/origin for OCP?